test/acceptance/WeeklyPromoSpec.scala

```diff
@@ -0,0 +1,350 @@
+      assert(promoPage.DestinationList.menuContainsDescription(
+        "UnitedKingdom",
+        0,
+        "£30 every 3 months")
+      )
+
+      And("the annual link should be correct")
+      assert(promoPage.DestinationList.menuContainsLink(
+        "UnitedKingdom",
+        1,
+        "/checkout/weeklyzonea-guardianweeklyannual?countryGroup=uk&promoCode=10ANNUAL")
+      )
+
+      And("the annual title should be correct")
+      assert(promoPage.DestinationList.menuContainsTitle(
+        "UnitedKingdom",
+        1,
+        "Annual")
+      )
+
+      And("the annual description should be correct")
+      assert(promoPage.DestinationList.menuContainsDescription(
+        "UnitedKingdom",
+        1,
+        "£108 for 1 year, then standard rate (£120 every year)")
+      )
+
+    }
+
+    scenario("ZA user lands on GW promo page", WeeklyPromoAcceptance) {
+      val promoCode = "/p/10ANNUAL"
+
+      When("The user is in South Africa")
+      val promoPage = WeeklyPromo(endpoint = promoCode, country = "ZA")
+
+      go.to(promoPage)
+      assert(promoPage.pageHasLoaded())
+
+      Then("South Africa should be the first list item")
+      assert(promoPage.DestinationList.firstElementAttribute("data-dropdown-menu") === "SouthAfrica")
+
+      When("they click for delivery to South Africa")
+      promoPage.DestinationList.chooseLocalDelivery()
+
+      Then("the correct menu should become visible")
+      assert(promoPage.DestinationList.menuIsVisible("SouthAfrica"))
+
+      And("the quarterly link should be correct")
+      assert(promoPage.DestinationList.menuContainsLink(
+        "SouthAfrica",
+        0,
+        "/checkout/weeklyzonec-guardianweeklyquarterly?countryGroup=us")
+      )
+
+      And("the quarterly title should be correct")
+      assert(promoPage.DestinationList.menuContainsTitle(
+        "SouthAfrica",
+        0,
+        "Quarterly")
+      )
+
+      And("the quarterly description should be correct")
+      assert(promoPage.DestinationList.menuContainsDescription(
+        "SouthAfrica",
+        0,
+        "US$65 every 3 months")
+      )
+
+      And("the annual link should be correct")
+      assert(promoPage.DestinationList.menuContainsLink(
+        "SouthAfrica",
+        1,
+        "/checkout/weeklyzonec-guardianweeklyannual?countryGroup=us&promoCode=10ANNUAL")
+      )
+
+      And("the annual title should be correct")
+      assert(promoPage.DestinationList.menuContainsTitle(
+        "SouthAfrica",
+        1,
+        "Annual")
+      )
+
+      And("the annual description should be correct")
+      assert(promoPage.DestinationList.menuContainsDescription(
+        "SouthAfrica",
+        1,
+        "US$234 for 1 year, then standard rate (US$260 every year)")
+      )
+
+    }
+
+    // NEW PRICE/PRODUCT TEST
+
+    scenario("UK user lands on Guardian Weekly promotion page AFTER 10:45 on 10/10/18", WeeklyPromoAcceptance) {
+      val promoCode = "/p/10ANNUAL"
```

Comment:
We could pick up coverage of [this change](https://github.com/guardian/subscriptions-frontend/pull/1165) by navigating to /weekly and allowing the redirect to choose the promo?

Reply:
The redirect kills the `gwoct18` param I think. But if not, then yes.

Reply:
Feel free to consider investigating/adding this in a separate PR though, the changes here are already providing a lot of value 👍

Reply:
Actually it seems to send it through. Looks a little weird but might just work. Let me try...